package.json

```diff
@@ -11,7 +13,14 @@
     "react-bootstrap": "^1.4.0",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
-    "web-vitals": "^0.2.4"
+    "web-vitals": "^0.2.4", 
+    "react-icons": "^3.11.0",
```

Comment:
It looks like you switched to using the font-awesome icons, so we can probably remove this  `react-icons` import, right?

Reply:
sounds good